pandas/_libs/period.pyx

```diff
@@ -31,8 +31,11 @@ from pandas._libs.tslib import Timestamp, iNaT, NaT
 from tslibs.timezones cimport (
     is_utc, is_tzlocal, get_utcoffset, get_dst_info, maybe_get_tz)
 
+from tslibs.frequencies cimport (
+    get_freq_code, get_base_alias, get_to_timestamp_base, _get_freq_str,
+    _get_rule_month)
+from tslibs.frequencies import _MONTH_NUMBERS
```

Comment:
can't this be a cimport?

Reply:
At the moment `_MONTH_NUMBERS` is not cdef'd.  It is imported by tseries.frequencies, but not used there.

Reply:
oh, then take it out